deployments/container/Makefile

```diff
@@ -84,6 +84,7 @@ $(IMAGE_TARGETS): image-%:
 		$(DOCKER_BUILD_PLATFORM_OPTIONS) \
 		--tag $(IMAGE) \
 		--build-arg GOLANG_VERSION="$(GOLANG_VERSION)" \
+		--build-arg TOOLKIT_VERSION_SHA="$(TOOLKIT_VERSION_SHA)" \
```

Comment:
Do we want to extract the information from the go modules? (or maybe we do).

Reply:
Yes, as mentioned in the PR description, that Is the long-term goal (though it is hardcoded for now).